include/coap2/uri.h

```diff
@@ -79,17 +82,36 @@ coap_uri_t *coap_clone_uri(const coap_uri_t *uri);
  * Parses a given string into URI components. The identified syntactic
  * components are stored in the result parameter @p uri. Optional URI
  * components that are not specified will be set to { 0, 0 }, except for the
- * port which is set to @c COAP_DEFAULT_PORT. This function returns @p 0 if
- * parsing succeeded, a value less than zero otherwise.
+ * port which is set to default port for the protocol. This function returns
```

Comment:
"to default port for the protocol" → "to the default port for the protocol"

Reply:
Corrected